Philippines ocd id update

GoogleCivics-Kantar opened the pull request:
HI @jpmckinney 

We have made the following changes for Philippines.

1. Added Alias.csv file where Maguindanao's 1st, Maguindanao's 2nd districts are renamed to Maguindanao del Norte, Maguindanao del Sur respectively.
2. We have added a new district (South Cotabato's 3rd) in congressional_district.csv file.
3. We have added valid-through dates to  Maguindanao's 1st, Maguindanao's 2nd districts.
4. Also we have added above changes in country-ph.csv

GoogleCivics-Kantar commented:
> For the Maguindanao renaming, shouldn't the Norte and Sur be validFrom 2025-05? As-is, it means all of 1st, 2nd, Norte and Sur are valid until then.

@jpmckinney  we have made the suggested change

jpmckinney commented:
It looks like there were more changes, and the Manguidanao changes might have already been completed? See Redistricting on Wikipedia: https://en.m.wikipedia.org/wiki/2025_Philippine_House_of_Representatives_elections

An official source would be good to verify against.

azuser commented on identifiers/country-ph/congressional_district.csv:
```diff
@@ -252,4 +252,7 @@ ocd-division/country:ph/cd:zamboanga_del_norte_3rd,Zamboanga del Norte's 3rd,,
\ No newline at end of file
+ocd-division/country:ph/cd:zamboanga_sibugay_2nd,Zamboanga Sibugay's 2nd,,
+ocd-division/country:ph/cd:south_cotabato_3rd,South Cotabato's 3rd,2025-05,
+ocd-division/country:ph/cd:maguindanao_del_norte,Maguindanao del Norte,2025-05,
```
The Maguindanao del Norte and del Sur have been represented in the House of Representatives since 2022. I would expect validFrom to be set to 2022.

GoogleCivics-Kantar replied:
Hi @azuser we have updated valid from date of Maguindanao del Norte and del Sur to 2022.

GoogleCivics-Kantar replied:
@azuser we have updated the full valid from date for Maguindanao del Norte and Maguindanao del Sur
and full valid through dates for Maguindanao's 1st and Maguindanao's 2nd.

azuser commented on identifiers/country-ph/congressional_district.csv:
Please arrange the list in alphabetical order.

azuser replied:
Can you please adjust the order of 
ocd-division/country:ph/cd:biñan,Biñan's at-large,,
ocd-division/country:ph/cd:biliran,Biliran's at-large,,

CivicsAbhishek commented on identifiers/country-ph/aliases.csv:
```diff
@@ -0,0 +1,3 @@
+id,sameAs,sameAsNote
```
@azuser Since this is case of redistricting, new districts should be created. Deleted the alias.csv as suggested. Kindly have a look in the same.

CivicsAbhishek commented:
> It looks like there were more changes to districts, and the Manguidanao changes might have already been completed? See Redistricting on Wikipedia: https://en.m.wikipedia.org/wiki/2025_Philippine_House_of_Representatives_elections
> 
> An official source would be good to verify against.

@jpmckinney In the 18th and 19th Congress, bills were proposed and passed for redistricting and establishing new provinces for Maguindanao del Norte and Maguindanao del Sur, as well as creating a new district for South Cotabato's 3rd congressional district. These changes impact national representation, while the reapportionment of Makati's 2nd congressional district and Taguig/Pateros's 1st and 2nd districts do not affect national representation, as it pertains to the local-level redistribution of barangays. Agusan del Norte's changes has been already done.

In 19th Congress, bills for redistricting have been proposed but were not passed.

Maguindanao's 1st and 2nd congressional districts were two of the congressional districts in the province of Maguindanao, Philippines till Sept 2022. In 2022, a plebiscite was held to divide Maguindanao and create the provinces of Maguindanao del Norte and Maguindanao del Sur, which involved extending the districts by incorporating cities. As a result of this reapportionment, Maguindanao's 1st and 2nd districts were dissolved.

GoogleCivics-Kantar commented:
@azuser we have made the following changes

1. We have added two new districts: Agusan del Norte's and Butuan in congressional_district.csv file.
2. We have added valid-through dates to Agusan del Norte's 1st, Agusan del Norte's 2nd districts.
3. Also, we have made the above changes in country-ph.csv.

jpmckinney reviewed (approved):
Okay, confirming that

- https://en.m.wikipedia.org/wiki/Maguindanao_del_Norte%27s_at-large_congressional_district and https://en.m.wikipedia.org/wiki/Maguindanao_del_Sur%27s_at-large_congressional_district are indeed active, and replace the old 1st and 2nd.
- These will be contested for the first time in 2025 (or for the first time since being re-created):
  - https://en.m.wikipedia.org/wiki/Agusan_del_Norte%27s_at-large_congressional_district (re-created), somewhat related to the former Agusan del Norte's 2nd
  - https://en.m.wikipedia.org/wiki/Legislative_district_of_Butuan, somewhat related to the former Agusan del Norte's 1st
  - https://en.m.wikipedia.org/wiki/South_Cotabato%27s_3rd_congressional_district
  
[General Santos](https://en.m.wikipedia.org/wiki/General_Santos%27s_at-large_congressional_district) mentioned on Wikipedia already exists in this file (and is already active). 

https://en.m.wikipedia.org/wiki/2025_Philippine_House_of_Representatives_elections indicates that there is indeed 1 "New districts from redistricting laws from previous Congress" which matches this change.
